apollo-federation/src/supergraph/mod.rs

```diff
@@ -1576,6 +1691,103 @@ lazy_static! {
     };
 }
 
+fn insert_directive(
+    schema: &mut FederationSchema,
+    pos: &CompositeTypeDefinitionPosition,
+    directive: Component<Directive>,
```

Comment:
Looks like a merge error here - the build is failing due to a missing `use` for `Directive`

Reply:
yep, thanks. i had fixed it just forgot to commit it!